community/jfsutils/depends

```diff
@@ -0,0 +1 @@
+libuuid
```

Comment:
And I think this is probably just a make dep - can you test that to confirm?

Reply:
confirmed